extensions/vertx-http/runtime/pom.xml

```diff
@@ -100,6 +100,14 @@
             <artifactId>mockito-junit-jupiter</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.graalvm.sdk</groupId>
+            <artifactId>nativeimage</artifactId>
```

Comment:
We usually mark this one as provided as I'm pretty sure we don't want the artifact around in JVM mode? Except if we ended up doing it in the parent or the BOM?

Reply:
You are right. I added through the IDE and didn't look back.